src/docfx/template/TemplateEngine.cs

```diff
@@ -205,41 +205,9 @@ private RenderType GetTocRenderType()
 
             var jsonSchema = JsonUtility.DeserializeData<JsonSchema>(schemaString, new FilePath(schemaFilePath));
 
-            // temporary mapping, will retired after we support config it in UI portal
-            jsonSchema = LearnErrorMapping(mime, jsonSchema);
-
             return new JsonSchemaValidator(jsonSchema, forceError: true);
         }
 
-        private static JsonSchema LearnErrorMapping(string mime, JsonSchema jsonSchema)
-        {
-            var mappingPath = mime switch
-            {
-                "LearningPath" => "data/schemas/learningpath-error-mapping.json",
```

Comment:
>"data/schemas/learningpath-error-mapping.json" [](start = 34, length = 46)

Does the content of these files transfer to buildValidationRules?

Reply:
not yet, so I keep mapping files.

Reply:
so the mapping will  be disable for learn validation, how about we transfer the content first.

Reply:
good idea. Do we have test case to make sure the transfer is correct?

Reply:
content transfered